src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1589,6 +1592,58 @@ def use_radamsa_mutator_plugin(extra_env):
   return True
 
 
+def use_peach_mutator(extra_env, arguments):
+  """Decide whether or not to use peach mutator, and set up all of the
+  environment variables necessary to do so."""
+  # TODO(mpherman): Include architecture info in job definition and exclude
+  #  i386.
+  if environment.platform() != 'LINUX' or environment.get_value(
+      'MEMORY_TOOL') != 'ASAN':
+    return False
+
+  # Select grammar.
+  grammar = fuzzer_utils.extract_argument(arguments, 'grammar')
```

Comment:
Could we move more of this logic into get_path? Essentially everything from here to through the log_error, returning None instead of false, then returning False from this function if not pit_path.

Reply:
Done